packages/ui/src/components/Form/customField/CustomLongTextField.tsx

```diff
@@ -1,9 +1,11 @@
-import { HTMLFieldProps } from 'uniforms';
 import { LongTextField } from '@kaoto-next/uniforms-patternfly';
+import { FunctionComponent, RefObject } from 'react';
 import './CustomLongTextField.scss';
 
-export type CustomLongTextFieldProps = HTMLFieldProps<string, HTMLDivElement>;
+type LongTextFieldProps = Parameters<typeof LongTextField>[0] & { ref: RefObject<HTMLTextAreaElement> };
```

Comment:
Since `LongTextFieldProps` is the type defined for `LongTextField` props and since we are adding a new property of `ref` to it. Giving it a new name should be the preference?

Reply:
`LongTextFieldProps` it's not exported from `uniforms-patternfly`, therefore we could use the name.